plugins/integration/providers/voicebase/VoicebaseClientHelper.php

```diff
@@ -94,7 +104,7 @@ private function sendAPICall($url, $options = null, $noDecoding = false)
 		if(!$noDecoding)
 		{
 			$stringResult = $result;
-			$result = json_decode($result);
+			$result = json_decode($result, true);
```

Comment:
Why changing to assoc arrays?

Reply:
@rkreich if the json object has depth (or changes to contain depth) the json_decode will fail.